src/common/src/config.rs

```diff
@@ -1145,6 +1145,10 @@ pub struct StreamingDeveloperConfig {
     /// even if session variable set.
     /// If true, it's decided by session variable `streaming_use_shared_source` (default true)
     pub enable_shared_source: bool,
+
+    #[serde(default = "default::developer::switch_jdbc_pg_to_native")]
+    /// When true, all jdbc sinks for pg will be switched to native pg sinks.
+    pub switch_jdbc_pg_to_native: bool,
```

Comment:
Why it's `switch_jdbc_pg_to_native`, instead of something like `use_native_jdbc_for_pg`? I don't quite get what does the `switch` mean here.

Reply:
It's to switch from jdbc pg sink to rust pg sink:
```
We support switching from connector='jdbc' for jdbc postgres sinks to connector='postgres'.
```

Meaning that the sink does not build and run on our java connector. Instead, we will run it on the rust postgres sink.

Reply:
Added more docs.